sdk/examples/client/03_simple_block.rs

```diff
@@ -8,7 +8,7 @@ use iota_sdk::client::{Client, Result};
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    dotenv::dotenv().ok();
+    dotenvy::dotenv().ok();
```

Comment:
Missing comment

Reply:
yea, good find. that's hard to automatically replace if it's not there in the first place :trollface: